app/views/customer_orders/index.html.erb

```diff
@@ -9,6 +9,8 @@
   <p><a href="/orders/<%= order.id %>"><%= order.id %></a>
     Special Instructions: <%= order.special_instructions %>
   </p>
+<%= link_to "Delete Order", "/orders/#{order.id}/delete" %>
```

Comment:
Looks like your link here is sending a `GET` request instead of `DELETE`, and you've configured your routes to call the delete method on that request. This works, but I added a bit at the end of this line so that it reads
`<%= link_to "Delete Order", "/orders/#{order.id}", method: :delete %>`

That sends an actual `DELETE` request instead, and you can update your routes to look for `delete '/orders/:id', to: 'orders#destroy'`

Both should work but I think this way is more ReSTful :)